### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by snoyberg** [dbp/hworker-ses#2]
This is blocking inclusion in Stackage.

```
Unpacking to hworker-ses-0.1.0.0/
Resolving dependencies...
Configuring hworker-ses-0.1.0.0...
Building hworker-ses-0.1.0.0...
Preprocessing library hworker-ses-0.1.0.0...
[1 of 1] Compiling System.Hworker.SES ( src/System/Hworker/SES.hs, dist/build/System/Hworker/SES.o )

src/System/Hworker/SES.hs:86:29:
    Not in scope: ‘getEnv’
    Perhaps you meant ‘newEnv’ (imported from Control.Monad.Trans.AWS)

src/System/Hworker/SES.hs:92:54:
    Not in scope: ‘bHtml’
    Perhaps you meant ‘bhtml’ (line 78)
```

**Comment by dbp** [comment]
@snoyberg This should be fixed with 0.1.1.0 (on hackage now). It now depends on amazonka >= 1 (with introduced the API changes that broke it).

**Comment by snoyberg** [comment]
Thanks, I'll reopen if the problem persists.
